localbuild/meta.yaml

```diff
@@ -72,6 +72,8 @@ requirements:
     - xstatic-bootstrap
     - pyperclip
     - geos <3.9.0
+    - sqlalchemy < 1.4.0
+    - sqlite < 3.35.1
 
```

Comment:
My local setup works with this too:

sqlite                    3.35.1               hdfb4753_0